timemachine/maps/estimators.py

```diff
@@ -0,0 +1,56 @@
+"""Utilities for computing mapped works or mapped u_kn matrix, incorporating invertible map_fxns"""
```

Comment:
Naming: `estimators.py` isn't quite right (doesn't contain estimators, but instead _adaptors_ for other estimators like EXP, BAR, MBAR that expect `works` array or `u_kn` matrix), but other naming ideas like `adaptors.py` didn't seem quite right either.